scripts/weaponEffects.js

```diff
@@ -169,6 +169,8 @@ const weaponEffects = {
     "mw_warp_rifle": "Warp Rifle",
     "mw_sherman_integrated": "Railgun",
     "default_tech_attack": "DefaultTech",
+    "lwfx_default_melee": "DefaultMelee",

```

Comment:
This is used when clicking the "Melee/Ranged" basic attack button. The basic attack is always marked as `attack_type "Melee"` -- so `lwfx_default_ranged` is probably never used.